fauxfactory/__init__.py

```diff
@@ -57,7 +57,7 @@ def _make_unicode(data):
         cast is necessary, a copy of ``data`` is returned.
 
     """
-    if sys.version_info.major == 2:
+    if sys.version_info[0] == 2:
```

Comment:
@Ichimonji10 actually it looks like ``sys.version_info`` returns a tuple "containing the five components of the version number: major, minor, micro, releaselevel, and serial" for both ``2.6.x`` and ``2.7.x`` **but** being able to access these values by name (i.e. ``sys.version_info.major``) can only be done with 2.7.x. If this is indeed the case the change above has merit.